Modules/_codecsmodule.c

```diff
@@ -68,6 +68,27 @@ _codecs_register(PyObject *module, PyObject *search_function)
     Py_RETURN_NONE;
 }
 
+/*[clinic input]
+_codecs.unregister
+    search_function: object
+    /
+
+Unregister a codec search function.
+
+If the search function is not registered, do nothing.
+[clinic start generated code]*/
+
+static PyObject *
+_codecs_unregister(PyObject *module, PyObject *search_function)
+/*[clinic end generated code: output=1f0edee9cf246399 input=8bfb99685e00d653]*/
+{
+    if (_PyCodec_Unregister(search_function)) {
```

Comment:
I prefer `< 0` to better explain that negative return value means an error.

```suggestion
    if (_PyCodec_Unregister(search_function) < 0) {
```

Reply:
done.